MdeModulePkg/Include/Protocol/MemoryProtectionSpecialRegion.h

```diff
@@ -0,0 +1,42 @@
+/** @file -- MemoryProtectionSpecialRegion.h
+
+  The memory protection special region protocol enables fetching all special regions
+  and adding new ones.
+
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#ifndef __MEMORY_PROTECTION_SPECIAL_REGION_PROTOCOL_H__

```

Comment:
can the __MEMORY_PROTECTION_SPECIAL_REGION_PROTOCOL_H__ get change to follow edk2 coding?
https://edk2-docs.gitbook.io/edk-ii-c-coding-standards-specification/v/release-2.20/5_source_files/53_include_files

Reply:
Updated the ifdef, and renamed the file to conform to the edk2 requirement of no duplicate file names